github_pr.py

```diff
@@ -21,11 +22,12 @@ class OwnerCannotShipError(Exception):
 class NoMergeCommentError(Exception):
     pass
 
+logger = logging.getLogger()
 
 def check_required_fields(required, **args):
     for i in required:
         if args[i] is None:
-            print 'Error: %s parameter not set' % i
+            logging.error(' %s parameter not set', i.upper())
```

Comment:
all of your `logging.<level>()` calls need to be `logger.<level>()`. Otherwise, they use the default logging config